source/codegen/metadata/nidigitalpattern/config.py

```diff
@@ -43,7 +43,7 @@
         }
     },
     'metadata_version': '2.0',
-    'module_name': 'nidigital',
+    'module_name': 'nidigitalpattern',
```

Comment:
We should record this change in CHANGES.md

Reply:
Done. I forgot to do it earlier.